interface/global-preferences.md

```diff
@@ -52,4 +52,8 @@ The `Reset Layout` button resets **Interface**s found in Incari to its default p
 ### Grid Color
 `Grid Color` sets the color of the *grid* found in the **ViewPort**. The color is set in the same way as `Bottom` and `Top Gradient Color` **Attributes**.
 
-`Reset colors` sets the `Bottom` and `Top Gradient Color` as well as the `Grid Color` of the **Viewport** back to their default colors. 
\ No newline at end of file
+`Reset colors` sets the `Bottom` and `Top Gradient Color` as well as the `Grid Color` of the **Viewport** back to their default colors. 
+
+In order to open the **Global Preference** **Interface**, please the see the illustration below:
```

Comment:
As far as I know, we are using the term **Module** to describe all of the following:
![image](https://user-images.githubusercontent.com/20472837/91955436-757c8100-ed03-11ea-8800-39c5bc93c885.png)
The word "interface" is used more for the overall GUI of Incari itself.